package.json

```diff
@@ -5,9 +5,9 @@
   "scripts": {
     "check:vanilla": "tsc --noEmit --project src/vanilla",
     "prestart:vanilla": "npm run build:vanilla",
-    "start:vanilla": "webpack-dev-server --config ./src/vanilla/webpack.config.babel.js --inline --hot --progress --https --port 8010",
+    "start:vanilla": " export SET NODE_OPTIONS=--openssl-legacy-provider && webpack-dev-server --config ./src/vanilla/webpack.config.babel.js --inline --hot --progress --https --port 8010",
```

Comment:
`export` is not really cross-platform. We should use `cross-env`.

Reply:
Also, in my case with Node 16, I get:

> node: --openssl-legacy-provider is not allowed in NODE_OPTIONS

Reply:
ok I'll switch to node 16

Reply:
done